packages/cli-kit/src/cli/api/graphql/admin/generated/types.d.ts

```diff
@@ -120,6 +120,67 @@ export type Scalars = {
   UtcOffset: {input: any; output: any}
 }
 
+/** Possible types of a metafield's owner resource. */
+export type MetafieldOwnerType =
```

Comment:
For context - are these generated automatically from the schemas via commands in the root `package.json`?